shared/chat/conversation/input/index.native.js

```diff
@@ -174,7 +174,10 @@ const Action = ({text, onSubmit, editingMessage, openFilePicker, isLoading}) =>
       </Text>
     </Box>
   ) : (
-    <Icon onClick={openFilePicker} type="iconfont-camera" style={styleActionButton} />
+    <Box style={styleActionButtonContainer}>
+      <Icon type="iconfont-mention" style={{...styleActionButton, paddingRight: 0}} />
```

Comment:
Noticed this but forgot to mention. Shouldn't this be in a future PR which makes this button work?

Reply:
Yeah it's implemented in the third PR actually (I got so caught up in handlers I almost forgot to put it in). I'm going to squash all three into this before merging and I'll probably mess up moving this to that PR 😅